applications/OptimizationApplication/custom_utilities/filtering/filter_function.cpp

```diff
@@ -27,19 +27,20 @@ FilterFunction::FilterFunction(const std::string& rKernelFunctionType)
 
     if (rKernelFunctionType == "gaussian") {
         // Type 1: Gaussian function
-        mFilterFunctional =  [](double radius, double distance) {return std::max(0.0, exp(-(distance*distance)));};
+        // at distance = radius, the filter value will be 1e-8
+        mFilterFunctional =  [](double radius, double distance) { return std::max(0.0, exp(-(8 * 2.3025850929940455 * distance * distance) / (radius * radius))); };
```

Comment:
where does this come from 2.3025850929940455  ?

Reply:
it is log_e(10)